matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/version/Versions.kt

```diff
@@ -60,7 +60,6 @@ private const val FEATURE_THREADS_MSC3771 = "org.matrix.msc3771"
 private const val FEATURE_THREADS_MSC3773 = "org.matrix.msc3773"
 private const val FEATURE_REMOTE_TOGGLE_PUSH_NOTIFICATIONS_MSC3881 = "org.matrix.msc3881"
 private const val FEATURE_EVENT_REDACTION_WITH_RELATIONS = "org.matrix.msc3912"
```

Comment:
I would ask to rename this flag `FEATURE_EVENT_REDACTION_WITH_RELATIONS` with `FEATURE_REDACTION_OF_RELATED_EVENT`

same remark for all method names and params
for example:
`withRelations` -> `withRelTypes`
`unstableWithRelations` -> `unstableWithRelTypes`